app/views/reviews/_review.html.haml

```diff
@@ -19,9 +19,11 @@
                 %span.nil_vote{ change_up_to: "1" }= link_to "+", votes_path(vote: { review_id: review.id, up: 1 }), method: :post
                 %span.voted{ change_up_to: "0" }= link_to "-", votes_path(vote: { review_id: review.id, up: 0 }), method: :post
 
-            %h5.review-title= review.title
-            %p= review.body
-            %p= "written by: #{review.user.username}"
+          .review-title-container.small-8
+            %h3.review-rating.small-1= "Rating: #{review.rating}"
+            %h5.review-title.small-7= review.title
+            %p.small-10= review.body
+            %p.small-12= "#{review.user.username}"
```

Comment:
`= "..."` should be rewritten as `...`<br>`%... \#{expression}` can be written without interpolation as `%...= expression`